tests/compose/test-misc-tweaks.sh

```diff
@@ -10,11 +10,13 @@ treefile_append "repos" '["test-repo"]'
 # test `recommends: true` (test-basic[-unified] test the false path)
 build_rpm foobar recommends foobar-rec
 build_rpm foobar-rec
+build_rpm quuz
+build_rpm corge
 
 echo gpgcheck=0 >> yumrepo.repo
 ln "$PWD/yumrepo.repo" config/yumrepo.repo
 # the top-level manifest doesn't have any packages, so just set it
-treefile_append "packages" '["foobar"]'
+treefile_append "packages" $'["\'foobar >= 0.5\' quuz \'corge < 1.0.0\'"]'
```

Comment:
Hmm, I think a more convincing example is building e.g. two `corge` packages, one at 1.0 and one at 2.0, then have the request be for `cargo < 2.0` and verify that `corge-1.0` is the one that got pulled in.